### DISCUSSION TITLE
Github student pack azure login problem

### DISCUSSION THREAD

**Discussion by asad4052** [community/community#23133] [New to GitHub]
<p>I have signed up for Github Student Developer Pack.<br>
Under benefits it says 'Get access by connecting your GitHub account on Microsoft Azure '.</p>
<p>I click through. And try to log in to Azure with my Github account. This is where I’m having a nightmare.</p>
<p>At first I get ‘It looks like you already have an account with us<br>
The email address in your GitHub profile belongs to your work or school. You cannot use GitHub to sign in. Choose Next to sign in with your work or school account credentials.’</p>
<p>I click next and try to sign in with my school account email. I get another error. It says email not recognised. So I try to create an email address. Now it says I can’t actually create an account for my school. The email can only be for personal use.</p>
<p>Please if anybody can help!!</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>For student benefits and related conversations, we recommend visiting the <a href="https://github.com/orgs/community/discussions/17814">Student Developer Pack FAQs post</a> as that may help answer your questions. If not, please use that Discussion to ask for assistance as that is the Education Community’s new home.</p>

**Reply by osamaraghib01** [reply]
please help me if your issue solved ? I am also i have the same issue @asad4052 @queenofcorgis

**Reply by MohammedSohebb** [reply]
even  I am facing the same issue,, please help

**Reply by AyushSehrawat** [reply]
Solved it by using my teachers mail id, if you can get one maybe use it.

**Comment by ghost** [comment]
@asad4052 Was your issue solved ? I am currently having the same issue.
Also @queenofcorgis `https://github.com/orgs/github-community/discussions/9` doesn't seem to work anymore

**Reply by queenofcorgis** [reply]
Thanks for letting me know, here is the updated link: https://github.com/orgs/community/discussions/17814

**Reply by AyushSehrawat** [reply]
This doesn't seem to help with the Azure issue (Also discussion is locked) . Nevermind, i will try to create support request on Azure again (failed for some reason last time).

**Comment by ExplosiveGeek77** [comment]
Yep, having the same issue stuck on the ‘It looks like you already have an account with us
The email address in your GitHub profile belongs to your work or school. You cannot use GitHub to sign in. Choose Next to sign in with your work or school account credentials.’ and when I do so, I get none of the benefits. I swear this process is so annoying

**Reply by Lemon711zzz** [reply]
我也遇到了同样的问题，和微软客服联系了三天，似乎找到了解决办法。
1.将你的Github邮箱绑定一个其他非教育或工作的邮箱，并且解绑你的教育邮箱。
2.登录Azure，使用Github方式登录。
3.如果成功的话就不会遇到这个问题了，但在之后的验证环节，选择验证方式的时候，不要选择以学校验证，而是使用github验证。
4.另外，我在步骤2之后还在Github中重新绑定了我的教育邮箱，并将他设置为主要邮箱。
希望能够帮到你

**Reply by nostorian** [reply]
@ExplosiveGeek77 did u manage to fix it?

**Reply by ExplosiveGeek77** [reply]
@fw-real Yes, I did actually. What I did was disconnect my school account from my
GitHub account and then sign into azure with GitHub. You don't lose the
benefits and then the website won't bother you about issues with your
school account. Hope this helps!

**Comment by nostorian** [comment]
same problem here. unable to redeem the azure benefits and facing the It looks like you already have an account with us issue

**Comment by aureliozenortjro** [comment]
If Microsoft accepts a partnership with GitHub, in theory it would be accepting the established criteria, if GitHub already validates the person, why does Microsoft invent more rules? It doesn't seem like a partnership to me.

(Português) Se a Microsoft aceita uma parceria com o GitHub, na teoria ela estaria aceitando os critérios estabelecidos, se o GitHub já faz a validação da pessoa, por que a Microsoft inventa mais regras? Não me parece uma parceria.
